src/olympia/addons/serializers.py

```diff
@@ -114,6 +121,20 @@ def get_review_url(self, obj):
     def get_icon_url(self, obj):
         return absolutify(obj.get_icon_url(64))
 
+    def get_theme_data(self, obj):
+        theme_data = None
+
+        if obj.type == amo.ADDON_PERSONA:
+            try:
+                if obj.persona is None or hasattr(obj.persona, '_broken'):
```

Comment:
I think you should comment that like you did below, either with the django ticket or with a reference to the comment below.